ui/jstests/test_manage_taxonomies.jsx

```diff
@@ -35,25 +69,152 @@ define(['QUnit', 'jquery', 'setup_manage_taxonomies', 'reactaddons',
       }
     ]
   };
-  function reportError(msg) {
-    if (msg) {
-      console.error(msg);
+  var vocabularies = [
+    {
+      "vocabulary": vocabulary,
+      "terms": vocabulary.terms
     }
+  ];
+  var listOfVocabularies = {
+    "count": 1,
+    "next": null,
+    "previous": null,
+    "results": [
+      {
+        "id": 1,
+        "slug": "difficulty",
+        "name": "difficulty",
+        "description": "easy",
+        "vocabulary_type": "f",
+        "required": false,
+        "weight": 2147483647
+      }
+    ]
+  };
+  var listOfTerms = {
+    "count": 2,
+    "next": null,
+    "previous": null,
+    "results": vocabulary.terms
+  };
+  var saveVocabularyResponse;
+  function  addTermsComponentRenderTest(assert, rootNode) {
```

Comment:
It looks like you are only asserting existing behavior in this function. Please rename to `assertTerms` or something similar instead, or change the function to add a term and assert the newly inserted term